pkg/apis/kubeone/types.go

```diff
@@ -923,6 +926,95 @@ type StaticAuditLogConfig struct {
 	LogMaxSize int `json:"logMaxSize,omitempty"`
 }
 
+type WebhookAuditLog struct {
+	// Enable
+	// Default value is false.
+	Enable bool `json:"enable,omitempty"`
+
+	// Config
+	Config WebhookAuditLogConfig `json:"config"`
+}
+
+type WebhookMode string
+
+const (
+	Batch          WebhookMode = "batch"
+	Blocking       WebhookMode = "blocking"
+	BlockingStrict WebhookMode = "blocking-strict"
```

Comment:
For a follow up:
```suggestion
	BatchWebhookAuditLogMode          WebhookAuditLogMode = "batch"
	BlockingWebhookAuditLogMode       WebhookAuditLogMode = "blocking"
	BlockingStrict WebhookAuditLogMode WebhookAuditLogMode = "blocking-strict"
```
or something like that